test/shrinkwrap.ts

```diff
@@ -467,3 +467,39 @@ test('repeat install with no inner shrinkwrap should not rewrite packages in nod
   const m = project.requireModule('is-negative')
   t.ok(m)
 })
+
+test('installing from shrinkwrap when using npm enterprise', async (t: tape.Test) => {
+  const project = prepare(t)
+
+  const opts = testDefaults({registry: 'https://npm-registry.compass.com/'})
+
+  await installPkgs(['is-positive@3.1.0'], opts)
+
+  const shr = await project.loadShrinkwrap()
+
+  t.deepEqual(shr, {
+    dependencies: {
+      'is-positive': '3.1.0'
+    },
+    packages: {
+      '/is-positive/3.1.0': {
+        resolution: {
+          integrity: 'sha1-hX21hKG6XRyymAUn/DtsQ103sP0=',
+          tarball: 'https://npm-registry.compass.com/i/is-positive/_attachments/is-positive-3.1.0.tgz'
```

Comment:
I do prefer the relative path, since the URL to our npm enterprise instance has not been stable. Meaning we started at just an IP address before we added a DNS and then we moved to more separated and specific sub-domains. Every time we did that, we had to completely regenerate the shrinkwrap.yaml.